builder/main.go

```diff
@@ -55,8 +55,6 @@ func main() {
 		println(err.Error())
 		os.Exit(buildpackapplifecycle.ExitCodeFromError(err))
```

Comment:
Seems like this needs to move too. Could we extract all of the runner-related logic into a separate method with `return`s instead of `os.Exit`?

Reply:
Sure, we went ahead and did that.